source/administration/notices.rst

```diff
@@ -0,0 +1,39 @@
+In-Product Notices
+==================
+
+Mattermost in-product notices keep users and administrators informed of the newest product improvements, features, and releases.
+
+.. contents::
+  :local:
+
+.. image:: ../../images/notices.png
```

Comment:
Just a note that this isn't displaying but we agreed to confirm once it's published and go from there.